Messenger/FBMApplication.m

```diff
@@ -10,6 +10,22 @@ - (void)sendEvent:(NSEvent *)event {
     return;
   }
   
+  // Ctrl + Tab
+    if (event.type == NSKeyDown && (event.modifierFlags & NSControlKeyMask) && event.keyCode == 48) {
```

Comment:
keyCode is hardware-specific. Also, this code has recently been replaced with a hardware-independent solution.